equinox-shipment/Domain/Domain.fsproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
2.0 will be fine!
also put on warnign level 5 like other templates

Reply:
Updated thanks!